src/v0/destinations/google_adwords_offline_conversions/utils.js

```diff
@@ -346,6 +346,26 @@ const populateUserIdentifier = ({ email, phone, properties, payload, UserIdentif
   }
   return copiedPayload;
 };
+
+/**
+ * remove redundant ids
+ * @param {*} conversionCopy
+ */
+const updateConversion = (conversion) => {
```

Comment:
const updateConversion = (conversion) => {
  const conversionCopy = cloneDeep(conversion);
  if (conversionCopy.gclid) {
    delete conversionCopy.wbraid;
    delete conversionCopy.gbraid;
  } else if (conversionCopy.wbraid && conversionCopy.gbraid) {
    throw new InstrumentationError("You can't use both wbraid and gbraid.");
  } else if (conversionCopy.wbraid || conversionCopy.gbraid) {
    delete conversionCopy.userIdentifiers;
  }
  return conversionCopy;
};

can we use this sort of, as all are mutually exclusive conditions?